inc/header-image.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * This is the template the loads the relevant hero image.
+ *
+ * @package music_oral_history
+ * @since 2.0.0
```

Comment:
If this file is being newly added to the theme with this PR, then this seems like the right value to use.

Reply:
I updated all existing files to what was there before in some files, version 1.1.11. All new files are since 2.0.0